src/apps/AppCenter/AppCenter.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { AppBar, AppView, NavigationBar, TabBar, Viewport } from '@aragon/ui'
+import {
+  AppBar,
+  AppView,
+  NavigationBar,
+  TabBar,
+  Viewport,
+  Button,
+} from '@aragon/ui'
 import MenuButton from '../../components/MenuPanel/MenuButton'
 import InstalledApps from './InstalledApps/InstalledApps'
 import DiscoverApps from './DiscoverApps/DiscoverApps'
 import UpgradeAppPanel from './UpgradeAppPanel'
 import EmptyBlock from './EmptyBlock'
 import { KERNEL_APP_BASE_NAMESPACE } from '../../aragonos-utils'
+import styled from 'styled-components'
```

Comment:
Nitpick here: could you move this `import` upwards near `react` or `prop-types`.
The convention we try to follow roughly goes like:
- External imports
- Internal components
- Internal utils